src/microbe_stage/editor/MicrobeEditorGUI.cs

```diff
@@ -795,6 +802,27 @@ public void UpdateAutoEvoResults(string results, string external)
         externalEffectsLabel.Text = external;
     }
 
+    public void ShowOrganelleMenu(OrganelleTemplate selectedOrganelle)
+    {
+        organelleMenu.SelectedOrganelle = selectedOrganelle;
+
+        // Disable delete/move option for nucleus or the last organelle.
+        if (selectedOrganelle.Definition.Name == nucleus.Name || editor.MicrobeSize < 2)
+        {
+            organelleMenu.EnableDeleteOption = false;
+            organelleMenu.EnableMoveOption = false;
+        }
+        else
+        {
+            organelleMenu.EnableDeleteOption = true;
+            organelleMenu.EnableMoveOption = true;
+        }
+
+        organelleMenu.RectPosition = GetViewport().GetMousePosition();
+        organelleMenu.Popup_();
```

Comment:
All the other methods positions the popup at the center of the screen. Using `Show` doesn't show the popup in modal form, though I found `ShowModal` which does that but I don't think there's any difference of just using Popup_

Reply:
So it works differently than the tutorial popup windows? Because `Show` works there?
If that's the case then styleguide needs a new guideline to: Not use Popup_ but prefer the explicit Popup* methods. If those don't perform as wanted, then use Show or ShowModal.

Reply:
I mean `Show` doesn't let the popup to automatically hide if there is a mouse click happening outside its area, so it kind of like defeats the purpose why a popup node is used in the first place because it's not taking advantage of the popup behaviour.

I use `Popup_` because it doesn't automatically position the popup at the center unlike the other PopupCentered methods (as I want the organelle menu to be placed at mouse pos anyway). I think `ShowModal` basically works the same as the Popup methods, except the latter emits `about_to_show` signal.

Reply:
WindowDialog is a child class of Popup. So you are stuck with Popup derived type in many cases.

I considered Using Popup_ for the tutorials, but found out that Show works much better.

Reply:
Alright, so I guess `ShowModal` should be used then.

Reply:
I'll update the issue https://github.com/Revolutionary-Games/Thrive/issues/1801 with the info that Show and ShowModal should be preferred over Popup_